kornia/filters/kernels.py

```diff
@@ -24,6 +24,98 @@ def gaussian(window_size, sigma):
     return gauss / gauss.sum()
 
 
+def gaussian_discrete_erf(window_size, sigma):
+    r"""discrete Gaussian by interpolating the error function.
+    adapted from:
+        https://github.com/Project-MONAI/MONAI/blob/master/monai/networks/layers/convutils.py
+    """
+    sigma = torch.as_tensor(sigma, dtype=torch.float, device=sigma.device if torch.is_tensor(sigma) else None)
```

Comment:
`torch.is_tensor` it's not supported with torchscript - I would rather use `isinstance(sigma, torch.Tensor)` as we have been using across the library and it's more pythonic :)